app/controllers/MyProjects/MediaController.php

```diff
@@ -469,7 +470,22 @@ public function save() {
 						$vs_upload_base_directory = $this->opo_item->getAppConfig()->get('upload_base_directory');
 						foreach($va_media_files_info as $vn_key => $va_media_file_info){
 							$va_errors_file = array();
-							$vs_tmp = "";
+
+                            //jquery-file-upload changes: overriding temp file and path                       
+                            $jfu_media_file_name = 'notset';
```

Comment:
The code style for collective access typically specifies type in variable names with a two-letter combinations, usually either 'v' (for standard variables) or 'p' (for URL parameter variables) as the first letter followed by a second letter indicating type ('s' string, 'b' boolean, 'n' for int/float, 'a' for array, etc.). So this variable name should be $vs_jfu_media_file_name.

Reply:
I will replace the var names. thanks for the tips.